cmd/jaeger/internal/integration/integration.go

```diff
@@ -38,7 +41,9 @@ type E2EStorageIntegration struct {
 // This function should be called before any of the tests start.
 func (s *E2EStorageIntegration) e2eInitialize(t *testing.T) {
 	logger, _ := testutils.NewLogger()
+	s.ConfigFile = createStorageCleanerConfig(t, s.ConfigFile)
```

Comment:
```suggestion
	configFile := createStorageCleanerConfig(t, s.ConfigFile)
```
it's better not to override the field in the struct, you only need this value in L49